lib/boxr/version.rb

```diff
@@ -1,3 +1,3 @@
 module Boxr
-  VERSION = "1.4.0"
+  VERSION = "1.4.1"
```

Comment:
I would make this 1.5.0 as it added new, non-breaking functionality rather than fixing a bug